PocketKit/Sources/Analytics/Tracker/PocketTracker.swift

```diff
@@ -40,7 +46,7 @@ public class PocketTracker: Tracker {
 }
 
 extension PocketTracker {
-    private func Event<T: Event>(from event: T) -> SelfDescribing? {
+    private func Event<T: OldEvent>(from event: T) -> SelfDescribing? {
```

Comment:
the method name should start lowercased.

Reply:
Not gonna rename, this is a legacy method.

Reply:
same as above, cool